### DISCUSSION TITLE
Allow the drilling down into the `other` dimension on apps.plugin

### DISCUSSION THREAD

**Discussion by hugovalente-pm** [netdata/netdata#12337] [Ideas]
This feature request, identified together with some users,  is to provide a means to drill down into the `other` dimension on the apps.plugin. This will provide the possibility to identify which processes are grouped inside that dimension. 

This ideally should provide a view of the impact on the metric being measured, e.g. if measuring `apps.cpu` and the `other` was having a usage of 15% we could have:
* 2.5 % on process1
* 8 % on process2
* 4.5 % on process3

Understanding that keeping time-series information over this could potentially be complex/not performant we could expose this for the current moment.

What do you guys think @cpipilas @stelfrag ?

**Comment by thiagoftsm** [comment]
@hugovalente-pm ,

We have `apps`, `cgroup` and `eBPF` working with PIDs to group them either as `apps groups` or `system service`. For we have something like this we would need:

1 - Centralize information from these plugins in one unique place.
2 - Expose this information for cloud. 
3 - We also need to understand that when we monitor software like database servers, the PIDs are created and closed in each connection. This implies to monitor not only PID, but also the PPID.

Everything I said about the common PIDs also applies to `other`, because `other` is a PID not listed inside `apps_cgroups.conf`.

Finally, I agree this is an interesting feature.

**Comment by stelfrag** [comment]
> Understanding that keeping time-series information over this could potentially be complex/not performant we could expose this for the current moment.

This is a nice idea and as you mention it will require some additional work. 

just "thinking" out loud 

The "other" dimension should also be accompanied by all the "processes" (or metrics) that are grouped (those should not automatically participate in any data queries) 

There needs to be another way to drill down on those (maybe an `expand` option in the data query) that may automatically recognize those "grouped" dimensions and add those in the result set (instead of the grouped one(s))

**Comment by hugovalente-pm** [comment]
@stelfrag to see if I understand your idea, you are suggesting that we can cross-reference somehow this "other" dimension and when getting more details for it we can query the metrics that will provide those details right?

this seems interesting and would achieve the purpose 😀
